apps/server/src/schemas/jsonSchema.ts

```diff
@@ -443,6 +443,7 @@ export const $Task: JSONSchema<S.Task> = {
   properties: {
     id: $Ulid,
     name: { type: 'string' },
+    groups: { type: 'array', items: { type: 'string' } },
   },
   required: ['id', 'name'],
```

Comment:
should: It might be worth adding `groups` to the required properties of a task - I struggle to see where we'd need it to be optional.

```suggestion
  required: ['id', 'name', 'groups'],
```